metaseq/model_parallel/criterions/vocab_parallel_cross_entropy.py

```diff
@@ -44,7 +44,7 @@ def forward(self, model, sample, reduce=True):
         loss = vocab_parallel_cross_entropy(net_output[0].float(), target)
         if has_pad:
             loss = loss * (target != self.padding_idx)
-        batch_loss = loss.mean(-1)
+        batch_loss = loss.sum(-1)  # TODO: length normalize?
```

Comment:
This seems to be an important change, what's the motivation?

Reply:
mean was added just in my previous PR. We need to see what makes most sense.